aikau/src/main/resources/alfresco/renderers/_ActionsMixin.js

```diff
@@ -144,7 +167,14 @@ define(["dojo/_base/declare",
        */
       addActions: function alfresco_renderers__ActionsMixin__addActions() {
          // Iterate over the actions to create a menu item for each of them...
-         if (this.customActions && this.customActions.length > 0)
+         if (this.mergeActions === true)
+         {
+            // Add the actions on the currentItem and then add additional custom actions...
+            this.currentItem.actions && array.forEach(this.currentItem.actions, lang.hitch(this, this.addAction));
```

Comment:
Could also be written `array.forEach(this.currentItem.actions, this.addAction, this);`, but makes no difference at all!!